integration/init_app/docker-asset/expected/.ship/release.yml

```diff
@@ -0,0 +1,16 @@
+---
+assets:
+  v1:
+    - docker:
+        image: busybox:latest
```

Comment:
+1 on andrew's comment re: helloscratch, busybox:latest is gonna change a lot

Reply:
didnt know about it. good point